bears/general/HTTPSBear.py

```diff
@@ -0,0 +1,85 @@
+import re
+import requests
+from urllib.parse import urlparse
+import requests
+
+from difflib import SequenceMatcher
+
+from bears.general.URLBear import URLBear
+from coalib.bears.LocalBear import LocalBear
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+from aenum import Flag
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+from coalib.bears.LocalBear import LocalBear
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.bearlib import deprecate_settings
+from coalib.settings.Setting import typed_list
+from coalib.results.HiddenResult import HiddenResult
+from coalib.results.SourceRange import SourceRange
+from coalib.parsing.Globbing import fnmatch
+from coalib.settings.Setting import typed_dict
+from coala_utils.decorators import (enforce_signature, generate_ordering,
+                                    generate_repr)
+
+# class HTTPSResult(HiddenResult):
+#     @enforce_signature
+#     def __init__(self, origin, affected_code,
+#                  link: str,
+#                  http_status_code: (int, None),
+#                  link_context: LINK_CONTEXT):
```

Comment:
commented code ;)

Reply:
@Makman2 Yeah still working on it :P I still have to write tests and get it to solve the problem on its own , also remove unnecessary heaers :P